torch/ao/quantization/qconfig_mapping.py

```diff
@@ -105,6 +105,10 @@ def _get_default_qconfig_mapping(is_qat: bool, backend: str, version: int) -> QC
         .set_object_type(torch.nn.functional.layer_norm, qconfig_layernorm) \
         .set_object_type(torch.nn.LayerNorm, qconfig_layernorm) \
 
+    if backend == 'onednn':
+        qconfig_mapping.set_object_type(torch.nn.LeakyReLU, qconfig) \
```

Comment:
do we support quantization for standalone leakyrelu module/op?

Reply:
I think it's required that the fused module shares the same `qconfig` with separate modules/ops otherwise there will be an error in UT. So, I added these here.

Reply:
OK, sounds good, we should be able to configure qconfig for patterns I think, but this will come up a bit later, cc @andrewor14 as FYI

Reply:
can you add a TODO comment here? so that we will remember to remove this later when we support fusion patterns in QConfigMapping

Reply:
OK, I have added a TODO comment in this PR: https://github.com/pytorch/pytorch/pull/89188